content/docs/concepts/architecture.md

```diff
@@ -0,0 +1,328 @@
+to influence how the {{<color "#008A00" >}}Endpoint{{< /color >}} provides service to the {{<color "#0050EF" >}}Client{{< /color >}}.
+
+A {{<color "#0050EF" >}}Client{{< /color >}} may be a:
+- Pod
+- VM
+- Physical Server
+
+### vWires
+
+That which connects a {{<color "#0050EF" >}}Client{{< /color >}} to an {{<color "#008A00" >}}Endpoint{{< /color >}} is a {{<color "#6A00FF" >}}vWire{{< /color >}} or {{<color "#6A00FF" >}}Virtual Wire{{< /color >}}.
+
+![vWire](/img/concepts/architecture/vWire.svg)
+
+The contract of a {{<color "#6A00FF" >}}vWire{{< /color >}} is:
+- A packet ingressing the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}} will egress at the {{<color "#008A00" >}}Endpoint{{< /color >}}
+- A packet ingressing the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#008A00" >}}Endpoint{{< /color >}} will egress the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}}
+- Only packets that ingressed the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}} will egress at the {{<color "#008A00" >}}Endpoint{{< /color >}}
+- Only packets that ingressed the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}} will egress the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#008A00" >}}Endpoint{{< /color >}}
+- An {{<color "#008A00" >}}Endpoint{{< /color >}} may have multiple incoming {{<color "#6A00FF" >}}vWires{{< /color >}}.
+- A {{<color "#0050EF" >}}Client{{< /color >}} may have multiple outgoing {{<color "#6A00FF" >}}vWires{{< /color >}}.
+- Each {{<color "#6A00FF" >}}vWire{{< /color >}} carries traffic for exactly one Network Service.
+
+In short, a {{<color "#6A00FF" >}}vWire{{< /color >}} acts like a virtual Wire between {{<color "#0050EF" >}}Client{{< /color >}} and {{<color "#008A00" >}}Endpoint{{< /color >}}.
+
+It should be noted that a {{<color "#0050EF" >}}Client{{< /color >}} *may* request the same Network Service multiple times, and thus have mutiple {{<color "#6A00FF" >}}vWires{{< /color >}} that happen to connect
+it to a particular {{<color "#008A00" >}}Endpoint{{< /color >}}.
+
+### Endpoints
+
+An {{<color "#008A00" >}}Endpoint{{< /color >}} in Network Service Mesh, sometimes called a Network Service {{<color "#008A00" >}}Endpoint{{< /color >}} or NSE is the 'thing' provides the Network Service to the {{<color "#0050EF" >}}Client{{< /color >}}.
+
+Network Service Mesh constructs a {{<color "#6A00FF" >}}vWire{{< /color >}} between the {{<color "#0050EF" >}}Client{{< /color >}} and the {{<color "#008A00" >}}Endpoint{{< /color >}}:
+
+![NSE](/img/concepts/architecture/nse.svg)
+
+An {{<color "#008A00" >}}Endpoint{{< /color >}} may be
+- a Pod running in the same K8s cluster
+- a Pod running in a different K8s cluster
+- a VM
+- an aspect of the physical network
+- Anything else to which packets can be delivered for processing
```

Comment:
This list does not look like a list in the web view.

Reply:
Fixed.